x-pack/plugins/ml/public/application/services/usage_collection.test.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { UsageCollectionSetup } from 'src/plugins/usage_collection/public';
+
+import { mlUsageCollectionProvider } from './usage_collection';
+
+describe('usage_collection', () => {
+  let usageCollection: jest.Mocked<UsageCollectionSetup>;
+
+  beforeEach(() => {
+    usageCollection = ({
+      reportUiCounter: jest.fn(),
+    } as unknown) as jest.Mocked<UsageCollectionSetup>;
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('mlUsageCollection', () => {
```

Comment:
nit: i think it would be clearer to rename this test (and `mlUsageCollection1`) to be more clear that they are for `exporting` and `importing`.

Reply:
good spot! those were meant to be temporary names

Reply:
Changed in https://github.com/elastic/kibana/pull/107772/commits/4fcdabb46850638ce3bc6f8c4f36acd6a335e89f